lib/librato.js

```diff
@@ -627,7 +630,7 @@ exports.init = function librato_init(startup_time, config, events, logger)
     }
     
     // Set host as a global tag
-    tags['host'] = sourceName;
+    tags['host'] = hostName;
```

Comment:
Is there a reason to use dotted notation like `measure.tags.source = measure.source` above, but not here?